configure.ac

```diff
@@ -79,8 +79,6 @@ AM_CONDITIONAL([PYTHON_SYNTAX_CHECK], [test "${enable_python_syntax_check}" = "y
 AC_CONFIG_FILES([
 	Makefile
 	bin/Makefile
-	bin/ovirt-ha-agent
-	bin/ovirt-ha-broker
```

Comment:
https://www.gnu.org/software/autoconf/manual/autoconf-2.68/html_node/Configuration-Files.html

So files mentioned here are the files you want to replace macros in it.
So valid to remove those here.